lib/grape-rabl/render.rb

```diff
@@ -1,8 +1,8 @@
 module GrapeRabl
   module Render
     def render(options = {})
-      env['api.endpoint'].options[:route_options][:rabl] = options.delete(:rabl) if options.include?(:rabl)
-      env['api.endpoint'].options[:route_options][:rabl_locals] = options.delete(:locals)
+      env['api.tilt.rabl'] = options.delete(:rabl) if options.include?(:rabl)
```

Comment:
This modifies `options`.  I don't think this is a side effect we want, justdo `options[:rabl]`, also use `key?` instead of `include?`.